### PR TITLE
add bootstrap-show-password v1.0.2 with git auto-upt auto-update, close #5590

### LINKED ISSUE #5590: [Request] bootstrap-show-password
**Library name:** bootstrap-show-password
**Git repository url:** https://github.com/wenzhixin/bootstrap-show-password
**npm package url(optional):** https://www.npmjs.com/package/bootstrap-show-password
**License(s):** MIT ??
**Official homepage:** https://github.com/wenzhixin/bootstrap-show-password
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by LeaYeh**
**git repo url:** https://github.com/wenzhixin/bootstrap-show-password/tree/1.0.2
**Watch 15
  Star 84
 Fork 33**
@Piicksarn  could u help me check this PR for #5590 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.

### CODE REVIEW

**Comment by Piicksarn** [comment]
@LeaYeh, the pr is fine, but need rebase to master.
And change the commit message to 
```
add bootstrap-show-password v1.0.2 with git auto-update.

close #5590  cc@author
```

**Comment by LeaYeh** [comment]
close this pr because https://github.com/cdnjs/cdnjs/commit/d0935dabf7069a8c693c02e99dcbf1f42bbfdd5f

**Comment by PeterDaveHello** [comment]
What's the relation between d0935da and this PR?

**Comment by LeaYeh** [comment]
@Piicksarn i reopen this pr, could you check this pr again?

**Comment by LeaYeh** [comment]
@Piicksarn ping

**Comment by Piicksarn** [comment]
I think the pr is fine!
@PeterDaveHello, would you mind to have a look at the pr?
Thank you!

**Comment by PeterDaveHello** [comment]
Commit message too long in the first line, not fine at all.

**Comment by LeaYeh** [comment]
@Piicksarn could you check it again?

File: ajax/libs/bootstrap-show-password/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "filename": "bootstrap-show-password.min.js",
+  "name": "bootstrap-show-password",
+  "version": "1.0.2",
+  "description": "Show/hide password plugin for twitter bootstrap.",
+  "author": {
+    "name": "zhixin wen",
+    "email": "wenzhixin2010@gmail.com",
+    "url": "http://wenzhixin.net.cn/"
+  },
+  "licenses": "MIT",
```
Comment (PeterDaveHello) [inline ajax/libs/bootstrap-show-password/package.json]: licenses?

Comment (LeaYeh) [inline ajax/libs/bootstrap-show-password/package.json]: sorry my mistake ><"

**Comment by Piicksarn** [comment]
@PeterDaveHello, i think this one is fine!
would you mind to have a look again?
Thank you!

**Comment by PeterDaveHello** [comment]
I thought I merged it, what's the problem?

**Pull request merged**